src/diffusers/models/unets/unet_2d_blocks.py

```diff
@@ -0,0 +1,3591 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
Why does this file only have 3591 lines but the original deleted `unet_2d_blocks.py` file has 3635 lines?